employees/tests/test_views.py

```diff
@@ -544,3 +545,67 @@ def _assert_response_contain_report(self, response, reports):
                     fields_to_check.append(getattr(report, field))
             for field in fields_to_check:
                 self.assertContains(response, field)
+
+
+class TestAuthorReportProjectView(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory()
+        self.project1 = ProjectFactory()
```

Comment:
If there is only one project please call it `self.project`.

Reply:
done